manual/C/ch_Finance-Quote.xml

```diff
@@ -211,7 +212,9 @@ SYNOPSIS
 
     <para>You have to register the <command>gnucash-cli</command> with a scheduler in order to get Online
       Quotes automatically and periodically. The method depends on your OS. For example, suppose you
-      want to update your file on 16:00 on every Friday.
+      want to update your file on 16:00 on every Friday. (Adjust the time to the close of the stock exchange 
+      and consider <emphasis>fund quotes</emphasis>, 
+      that the net asset value (NAV) is determined well after the market close).
```

Comment:
markets closed

Easy if only the regional market. Not sure about a fund investing in emerging markets around the world.

Have we already an glossary enty for NAV to link it?

Reply:
How can I link to the glossary? "Net assets" is present as an entry, not as an abbreviation NAV

Reply:
The types have their description in Tools->Prize Database [C/gnucash-help/tool-price.html].